BizsolERP/BizsolERPMain/Controllers/HomeController.cs

```diff
@@ -15,7 +15,7 @@ public HomeController(ILogger<HomeController> logger)
 
         public IActionResult Index()
         {
-            return View();
+            return View();/// mamisjsjklaisks sajssisj 
         }
 
```

Comment:
whay this here
![Screenshot 2023-05-04 115331](https://github.com/user-attachments/assets/c70b173f-cbe5-4b31-9893-b9b7c26f3484)

Reply:
done